src/azul/__init__.py

```diff
@@ -1652,9 +1652,13 @@ def google_oauth2_client_id(self) -> str | None:
     def monitoring_email(self) -> str:
         return self.environ['AZUL_MONITORING_EMAIL']
 
-    @property
-    def cloudwatch_dashboard_template(self) -> str:
-        return f'{config.project_root}/terraform/cloudwatch_dashboard.template.json.py'
+    def cloudwatch_dashboard_template(self, name: str) -> str:
+        valid_names = ['indexer']
+        if config.enable_mirroring:
+            valid_names.append('mirror')
+        assert name in valid_names, R('Invalid dashboard name', name)
+        return (f'{config.project_root}/terraform/'
+                f'cloudwatch_dashboard.{name}.template.json.py')
```

Comment:
I think extracting a variable here is a bit nicer to read
```suggestion
        file_name = f'cloudwatch_dashboard.{name}.template.json.py'
        return f'{config.project_root}/terraform/{file_name}'
```